packages/compass-e2e-tests/helpers/commands/run-find-operation.js

```diff
@@ -0,0 +1,129 @@
+const Selectors = require('../selectors');
+
+async function setFilter(client, tabName, value) {
+  await client.setAceValue(Selectors.queryBarOptionInputFilter(tabName), value);
+}
+
+async function setProject(client, tabName, value) {
+  await client.setAceValue(
+    Selectors.queryBarOptionInputProject(tabName),
+    value
+  );
+}
+
+async function setSort(client, tabName, value) {
+  await client.setAceValue(Selectors.queryBarOptionInputSort(tabName), value);
+}
+
+async function setCollation(client, tabName, value) {
+  await client.setAceValue(
+    Selectors.queryBarOptionInputCollation(tabName),
+    value
+  );
+}
+
+async function setMaxTimeMS(client, tabName, value) {
+  const selector = Selectors.queryBarOptionInputMaxTimeMS(tabName);
+  await client.clickVisible(selector);
+  await client.setValue(selector, value);
+}
+
+async function setSkip(client, tabName, value) {
+  const selector = Selectors.queryBarOptionInputSkip(tabName);
+  await client.clickVisible(selector);
+  await client.setValue(selector, value);
+}
+
+async function setLimit(client, tabName, value) {
+  const selector = Selectors.queryBarOptionInputLimit(tabName);
+  await client.clickVisible(selector);
+  await client.setValue(selector, value);
+}
+
+async function runFind(client, tabName) {
+  await client.clickVisible(Selectors.queryBarApplyFilterButton(tabName));
+}
+
+async function isOptionsExpanded(client, tabName) {
+  // it doesn't look like there's some attribute on the options button or container that we can easily check, so just look for a field that exists if it is expanded
```

Comment:
This might be a bit nitpicky, but can you wrap this comment? Event github doesn’t display it fully anymore :)

Reply:
OK. Weird that prettier didn't do that.